src/vector.rs

```diff
@@ -713,4 +718,26 @@ mod tests {
         assert_nearly_eq!(vector.x, rotated_vector.x);
         assert_nearly_eq!(vector.y, rotated_vector.y);
     }
+
+    #[test]
+    fn negative_works_with_default_vector() {
```

Comment:
the correct terminology is [zero vector](http://mathworld.wolfram.com/ZeroVector.html)

Reply:
c1144e5